src/hokusai/hokusai.yml

```diff
@@ -61,16 +61,25 @@ commands:
     parameters:
       filename:
         type: string
-        default: ./hokusai/test.yml
+        default: ""
```

Comment:
@zephraph I somehow lost or deleted you comment but check out the `when` [conditional](https://circleci.com/docs/2.0/configuration-reference/#the-when-step-requires-version-21)  if it's not provided it 
should run the `unless step` and just invoke `hokusai test`

Reply:
Yep, I deleted because I noticed what you did afterwards 😄